sal_interfaces/PMD/PMD_Telemetry.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsl"?>
+<SALTelemetrySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsd">
+  <SALTelemetry>
+    <Subsystem>PMD</Subsystem>
+    <EFDB_Topic>PMD_position</EFDB_Topic>
+    <item>
+      <EFDB_Name>position</EFDB_Name>
+      <Description>The position of the device.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>millimeter</Units>
```

Comment:
Please consider using `double` instead of `long` all positions in this XML, for several reasons:
* If this is a general purpose CSC then some devices you read may have floating-point values
* You can output `NaN` to indicate "no measurement available".

The only time this would be a bad idea is if the resolution was so fine that a double was not sufficient. That seems very unlikely for a micrometer. (In fact `float` might suffice, but I'd hate to lose information if it did not).